### PR TITLE
Update krona version and add nf-test

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
Related to https://github.com/nf-core/taxprofiler/issues/403

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by jfy133 — CHANGES_REQUESTED** [review]
Ther are krakentools/combinekreports changes in here, is that on purpose?

**Comment by LilyAnderssonLee** [comment]
> Ther are krakentools/combinekreports changes in here, is that on purpose?

I forgot to add stub run in this PR https://github.com/nf-core/modules/pull/6120

File: modules/nf-core/krona/ktimporttaxonomy/tests/main.nf.test.snap
Comment (sateeshperi) [inline modules/nf-core/krona/ktimporttaxonomy/tests/main.nf.test.snap:3]: looks like the assertions are not working ?

File: modules/nf-core/krona/ktimporttext/tests/main.nf.test.snap
Comment (sateeshperi) [inline modules/nf-core/krona/ktimporttext/tests/main.nf.test.snap:11]: same here....content is null

File: modules/nf-core/krona/ktimporttaxonomy/tests/main.nf.test.snap
Comment (LilyAnderssonLee) [reply modules/nf-core/krona/ktimporttaxonomy/tests/main.nf.test.snap]: Updated!

**Review by sateeshperi — COMMENTED** [review]
lgtm but, I am getting back to reviews after a hiatus so will wait for others to approve :)

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Good to go @LilyAnderssonLee !

**Pull request merged**
